recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,11 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  run:
+    - kraken ==0.10.5beta
```

Comment:
This is because when kraken does a major version change , it is unlikely it will still use jellyfish1. Therefore we should we cautious and do a strict version lock.